src/newSeed/stage2.ts

```diff
@@ -1,5 +1,21 @@
 import { BaseStage } from "newSeed/baseStage";
 
 export class Stage2 extends BaseStage {
+  proceed(): void {
+    let message: string;
 
+    if (!this.seedConfig?.projectDetails?.summary) {
+      message = "Please fill the summary of the project";
```

Comment:
Instead of "Please fill" make it "Please provide" or "Please enter".  Make sure all of the stages are consistent in how we word these validation messages.